web/templates/user/show.html.eex

```diff
@@ -1,31 +1,64 @@
-<h2>Show user</h2>
```

Comment:
In case the coordinator is checking a student record, She should be allowed to see the student_id and other records you hide from student or other users